cnd/src/config/settings.rs

```diff
@@ -267,4 +268,37 @@ mod tests {
                 listen: vec!["/ip4/0.0.0.0/tcp/9939".parse().unwrap()],
             })
     }
+
+    #[test]
+    fn settings_serializes_correctly() {
+        let file = File::default();
+        let settings = Settings::from_config_file_and_defaults(file).unwrap();
+
+        // Note: toml library serializes Network without a trailing new line?
+        let want = r#"[network]
+listen = ["/ip4/0.0.0.0/tcp/9939"]
+[http_api.socket]
+address = "0.0.0.0"
+port = 8000
+
+[http_api.cors]
+allowed_origins = "none"
+
+[database]
+sqlite = "/home/tobin/.local/share/comit/cnd.sqlite"
```

Comment:
This is not going to work on any machine other than yours :sweat_smile:

Reply:
Yeah I laughed at myself when I first saw the build failure.